kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/TestEngineLauncher.kt

```diff
@@ -192,6 +191,7 @@ class TestEngineLauncher(
    }
 
    fun withJs(): TestEngineLauncher = withPlatform(Platform.JS)
+   fun withWasmJs(): TestEngineLauncher = withPlatform(Platform.WasmJs).withTeamCityListener()
```

Comment:
Why do we add the `TeamCityListener` here, and only for wasm?

Reply:
It is remarkably complicated: The TeamCityListener formats test output in the "TeamCity" format. While on the JS platform, the same is accomplished by the Kotlin test infra (via Mocha), this is not the case for wasmJs (which does not use Mocha). Without the extra TeamCityListener, `gradlew test` in `kotest:kotest-framework:kotest-framework-multiplatform-plugin-gradle` fails. For details, see [this Slack thread](https://slack-chats.kotlinlang.org/t/16155931/adding-a-wasm-target-to-kotest-i-m-observing-strange-differe).